Source/WebCore/crypto/algorithms/CryptoAlgorithmX25519.cpp

```diff
@@ -87,12 +87,20 @@ void CryptoAlgorithmX25519::deriveBits(const CryptoAlgorithmParameters& paramete
         return;
     }
 
+    // Return an empty string doesn't make much sense, but truncating either at all.
+    // https://github.com/WICG/webcrypto-secure-curves/pull/29
+    if (length && !(*length)) {
```

Comment:
super nit:
```suggestion
    if (!length.value_or(0)) {
```

Reply:
ohh. right.. then what you have is perfect.
To re-state, Truncate only if length is explicitly provided!